src/redux/cofigureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, applyMiddleware } from 'redux';
```

Comment:
- [ ] Please import and use `combineReducers` even if you have just one reducer, as it is a project requirement 🚀

Reply:
Thanks, @helman101, I've forgotten to import that, my bad. I will do it now!